src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/cypress/e2e/ui/newCertifications/newECE5YearCertification.cy.ts

```diff
@@ -0,0 +1,152 @@
+import selectors from "../../../support/selectors";
+
+// Define the DeviceConfig interface.
+interface DeviceConfig {
+  device: string;
+  width: number;
+  height: number;
+}
```

Comment:
is there a way to define a global interface? this way we dont' need to declare it on every test.